src/explore-education-statistics-admin/src/services/dashboard/service.ts

```diff
@@ -17,22 +12,16 @@ export interface DashboardService {
 
 const service: DashboardService = {
   getMyThemesAndTopics(): Promise<ThemeAndTopics[]> {
-    return apiClient.then(client => client.get<ThemeAndTopics[]>('/me/themes'));
+    return client.get<ThemeAndTopics[]>('/me/themes');
   },
   getMyPublicationsByTopic(
     topicId: string,
   ): Promise<AdminDashboardPublication[]> {
-    return apiClient.then(client =>
-      client
-        .get<AdminDashboardPublication[]>('/me/publications', {
-          params: { topicId },
-        })
-        .then(publications =>
-          publications.map(publication =>
-            polyfill(publication, publicationPollyfilla),
-          ),
-        ),
-    );
+    return client
+      .get<AdminDashboardPublication[]>('/me/publications', {
```

Comment:
I'm wondering if we should drop the `me` from this endpoint as it would clean up the publications controller a bit?

Reply:
Thanks @jamescutts - @richardpopplehive possibly you have a thought?  I'd settle for /publications/in-progress or something along those lines?